### PR TITLE
AlertView Typescript Conversion

### PR DESCRIPTION
**Pull request by furkanarabaci**

### CODE REVIEW

File: ui/alertview/alertview.ios.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+enum MethodNames {
+  onDismiss = 'onDismiss'
+}
+
+export default class AlertViewIOS extends NativeComponent implements IAlertView {
+  private delegate: (method: { name: string }) => void;
+  private _onDismiss: (alertView: AlertViewIOS) => void;
+  android = {};
+  constructor(params: Partial<AlertViewIOS> = {}) {
```
Comment (cenkce) [inline ui/alertview/alertview.ios.ts:12]: `Partial<AlertViewIOS>` => `Partial<IAlertView>`

File: ui/alertview/alertview.ts [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+        top: number;
+        right: number;
+        bottom: number;
+      };
+    }>;
+  }): void;
+}
+
+export declare class AbstractAlertView implements IAlertView {
+  constructor(params?: Partial<AbstractAlertView>);
```
Comment (cenkce) [inline ui/alertview/alertview.ts:200]: `Partial< AbstractAlertView >` => `Partial<IAlertView>`

File: ui/alertview/alertview.android.ts [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+export default class AlertViewAndroid extends NativeComponent implements IAlertView {
+  private _android: AbstractAlertView['android'];
+  private __didSetOnDismissListener = true;
+  private __buttonCallbacks: Array<() => void> = [];
+  private __title = '';
+  private __message = '';
+  private __textBoxes: TextBox[];
+  private __onDismiss: AbstractAlertView['onDismiss'];
+  private _cancellable: AbstractAlertView['android']['cancellable'];
+  constructor(params: Partial<AlertViewAndroid> = {}) {
```
Comment (cenkce) [inline ui/alertview/alertview.android.ts:18]: `Partial< AlertViewAndroid >` => `Partial<IAlertView>`

**Review by cenkce — CHANGES_REQUESTED** [review]

File: ui/alertview/alertview.ios.ts (hunk 1 above)
Comment (furkanarabaci) [reply ui/alertview/alertview.ios.ts]: fixed

File: ui/alertview/alertview.ts (hunk 2 above)
Comment (furkanarabaci) [reply ui/alertview/alertview.ts]: fixed

File: ui/alertview/alertview.android.ts (hunk 3 above)
Comment (furkanarabaci) [reply ui/alertview/alertview.android.ts]: fixed

File: ui/alertview/alertview.ts [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+export declare class AbstractAlertView implements IAlertView {
+  constructor(params?: Partial<IAlertView>);
+  isShowing(): void;
+  show(): void;
+  dismiss(): void;
+  toString(): string;
+  title: string;
+  message: string;
+  onDismiss: (alertView?: AbstractAlertView) => void;
```
Comment (cenkce) [inline ui/alertview/alertview.ts:207]: AbstractAlertView yerine IAlertView kullanalim

**Review by cenkce — CHANGES_REQUESTED** [review]

File: ui/alertview/alertview.ts (hunk 4 above)
Comment (furkanarabaci) [reply ui/alertview/alertview.ts]: fixed

**Pull request merged**